testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/ServerAuthenticationTestCase.java

```diff
@@ -141,81 +127,47 @@ private void addUser(final File domainConfigDir, final String realm, final Strin
                         .getBytes(StandardCharsets.UTF_8));
     }
 
-    private CallbackHandler createCallbackHandler(final String username, final String password) {
-        return new CallbackHandler() {
-            public void handle(Callback[] callbacks) throws IOException, UnsupportedCallbackException {
-                for (Callback current : callbacks) {
-                    if (current instanceof NameCallback) {
-                        NameCallback ncb = (NameCallback) current;
-                        ncb.setName(username);
-                    } else if (current instanceof PasswordCallback) {
-                        PasswordCallback pcb = (PasswordCallback) current;
-                        pcb.setPassword(password.toCharArray());
-                    } else if (current instanceof RealmCallback) {
-                        RealmCallback rcb = (RealmCallback) current;
-                        rcb.setText(rcb.getDefaultText());
-                    } else {
-                        throw new UnsupportedCallbackException(current);
-                    }
-                }
-            }
-        };
-    }
-
-    private static ModelNode getLocalAuthPath(final String host) {
-        PathAddress path = PathAddress.pathAddress(HOST, host)
+    private static PathAddress getLocalAuthPath(final String host) {
+        return PathAddress.pathAddress(HOST, host)
                 .append(CORE_SERVICE, MANAGEMENT)
                 .append(SECURITY_REALM, "ManagementRealm")
                 .append(AUTHENTICATION, LOCAL);
-        return path.toModelNode();
     }
 
     private static void removeLocalAuth(final DomainClient client, final String host) throws Exception {
         // remove local auth, requires restart
-        final ModelNode op = new ModelNode();
-        op.get(OP).set(REMOVE);
-        op.get(OP_ADDR).set(getLocalAuthPath(host));
+        final ModelNode op = Util.createEmptyOperation(REMOVE, getLocalAuthPath(host));
         final ModelNode result = client.execute(op);
         assertEquals(result.toJSONString(true), SUCCESS, result.get(OUTCOME).asString());
```

Comment:
Yeah, I thought about removing this, but I was feeling a bit conservative and figured it'd be a fast check anyway, so why not. You're right though, its not needed.